Bio/PDB/parse_pdb_header.py

```diff
@@ -77,7 +87,22 @@ def _format_date(pdb_date):
         "Nov",
         "Dec",
     ]
-    month = str(all_months.index(pdb_date[3:6]))
+    month_name = pdb_date[3:6]
+
+    try:
+        month = "%02d" % all_months.index(month_name)
+    except ValueError:
+        if not permissive:
+            raise ValueError(
+                f"Non-standard month in PDB header: {month_name}."
+            ) from None
+
+        warnings.warn(
```

Comment:
I still object against warnings, as by default they get printed only once in one Python session.